hub/executors/indexers/vector/annoy/manifest.yml

```
@@ -0,0 +1,9 @@
+name: annoy
+description: an image with the indexer annoy
+author: Jina AI Dev-Team (dev-team@jina.ai)
```

We'd love to give credits to you! Please have your name typed here. ;)
```suggestion
author: Jina AI Dev-Team (dev-team@jina.ai)
```